### ISSUE TITLE
Defog lowers evasion on mon behind Substitutes

### ISSUE THREAD

**Issue by hedara90** [rh-hideout/pokeemerald-expansion#5386]
### Description

With `GEN_LATEST` settings for Defog, using it against a pokemon behind a Substitute while it has something that Defog can remove should not lower their Evasion.

Follow up on #5381

### Version

upcoming (Edge)

### Discord contact info

hedara

**Comment by AlexOn1ine** [comment]
> [...] using it against a pokemon behind a Substitute while it has something that Defog can remove should not lower their Evasion.

And if it doesn't have anything that defog can remove it lowers Evasion?

**Comment by i0brendan0** [comment]
> > [...] using it against a pokemon behind a Substitute while it has something that Defog can remove should not lower their Evasion.
> 
> And if it doesn't have anything that defog can remove it lowers Evasion?

No. As per Bulbapedia, lowering evasion of the target behind a substitute is only a Gen 4 thing. Therefore, if Defog's configuration is not Gen 4, it shouldn't lower the evasion of a target behind a substitute, regardless of field effects. 
Tangent: I think the field effect portion and the evasion portion should be 2 separate configurations but I might be alone on that.
